test/FunctionalTests/Tests/DataOData/Tests/OData.Query.TDD.Tests/TestUtilities/HardCodedTestModel.cs

```diff
@@ -747,6 +753,10 @@ internal static IEdmModel GetEdmModel()
                     throw new Exception("edmx:refernece must have a valid url." + uri.AbsoluteUri);
                 }, out parsedModel, out errors))
                 {
+                    (parsedModel.ReferencedModels.ElementAt(3).FindDeclaredType("Fully.Qualified.Namespace.Person") as IEdmEntityType).AddAlternateKey(parsedModel, new Dictionary<string, IEdmProperty>()

```

Comment:
Don't add the annotation using function. Please add the alternate key annotation in the following XML string.